lib/twitter-ads/http/ton_upload.rb

```diff
@@ -53,21 +53,21 @@ def initialize(client, file_path, opts = {})
     #
     # @return [String] The upload location provided by the TON API.
     def perform
-      if @file_size < MIN_FILE_SIZE
+      if @file_size < MAX_FILE_SIZE
         resource = "#{DEFAULT_RESOURCE}#{@bucket}"
         response = upload(resource, File.read(@file_path))
         response.headers['location'][0]
       else
         response   = init_chunked_upload
-        chunk_size = response.headers['x-ton-min-chunk-size'][0].to_i
+        chunk_size = response.headers['x-ton-max-chunk-size'][0].to_i
```

Comment:
The typical response is below.
This says we have 50 times limitation for requesting api in 15 minutes.

```
{"connection"=>["close"],
"content-length"=>["0"],
"content-type"=>["text/plain"],
"date"=>["Mon, 06 Nov 2017 16:50:51 GMT"],
"expires"=>["Mon, 13 Nov 2017 16:50:51 UTC"],
"location"=>["/1.1/ton/data/ta_partner/111111111111/xxxxxxxxxxxx.txt?resumable=true&resumeId=1111111"],
"server"=>["tsa_a"],
"set-cookie"=>["personalization_id=\"v1_Iq+Tb...Wed, 06 Nov 2019 16:50:51 UTC; Path=/; Domain=.twitter.com"],
"strict-transport-security"=>["max-age=631138519"],
"x-connection-hash"=>["xxxxxxxxxxxxxxxxxxxxxxxx"],
"x-content-type-options"=>["nosniff"],
"x-rate-limit-limit"=>["50"],
"x-rate-limit-remaining"=>["49"],
"x-rate-limit-reset"=>["1509987951"],
"x-response-time"=>["20"],
"x-ton-max-chunk-size"=>["67108864"],
"x-ton-min-chunk-size"=>["1048576"]}
```